cfme/fixtures/service_fixtures.py

```diff
@@ -12,23 +12,43 @@
 from cfme.infrastructure.provider import InfraProvider
 from cfme.rest.gen_data import dialog as _dialog
 from cfme.rest.gen_data import service_catalog_obj as _catalog
+from cfme.rest.gen_data import service_templates_rest as _service_templates
```

Comment:
Question: Why the name changes for `service_catalog_obj` and `service_templates_rest`?

Reply:
@john-dupuy I think that is because that's the standard that we have been using while calling any rest function from `gen_data`, just to distinguish it as a rest function.